spec/system/users_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+#:focus => true
+
+RSpec.feature "Users", type: :system do
+  describe 'ログイン前' do
+    let(:user) { build(:user) }
+    # 登録済みのメールアドレスが必要なため
+    let!(:other_user)  { create(:user, email: 'other@co.jp') }
```

Comment:
存在しないドメインである`example.com`を使いましょう。
```suggestion
    let!(:other_user)  { create(:user, email: 'other@example.com') }
```

Reply:
5df3657